index.js

```diff
@@ -223,11 +223,12 @@
           var fn = this.__outsideClickHandler;
           if (typeof document !== 'undefined') {
             var events = this.props.eventTypes || DEFAULT_EVENTS;
+            var passive = !this.props.preventDefault;
```

Comment:
mostly the fact that events are not cleanly passable things. In any compositional chain there is no guarantee that code won't have done something to the event before the user ever gets their hands on them, so making them *think* they're in control when they aren't is not setting up good user expectation, and I'd like to avoid it.

I think the real solution I'd like to see here is one that adds an exception specifically *only* for touch events. If those are the root cause of the problem, then using "not-preventDefault means it *must* be passive" logic is acceptable, but certainly not for every event, most of which won't ever be passive because they have *nothing* to do with page scrolls (the reason why the passive event concept was invented)